pavement.py

```diff
@@ -89,7 +89,7 @@
         GIT_REVISION = "Unknown"
 
     if not setup_py.ISRELEASED:
-        FULLVERSION += '.dev-' + GIT_REVISION[:7]
+        FULLVERSION += '.dev0+' + GIT_REVISION[:7]
```

Comment:
It's consistent with the `setup.py` file. Might be tricky to test on account of the paver dependency.